src/libs/actions/PaymentMethods.js

```diff
@@ -24,17 +47,41 @@ function getPaymentMethods() {
         excludeNotActivated: true,
     })
         .then((response) => {
+            // Convert bank accounts/cards from an array of objects, to a map with the bankAccountID as the key
+            const bankAccounts = _.object(_.map(lodashGet(response, 'bankAccountList', []), bankAccount => [bankAccount.bankAccountID, bankAccount]));
+            const debitCards = _.object(_.map(lodashGet(response, 'fundList', []), fund => [fund.fundID, fund]));
             Onyx.multiSet({
                 [ONYXKEYS.IS_LOADING_PAYMENT_METHODS]: false,
                 [ONYXKEYS.USER_WALLET]: lodashGet(response, 'userWallet', {}),
-                [ONYXKEYS.BANK_ACCOUNT_LIST]: lodashGet(response, 'bankAccountList', []),
-                [ONYXKEYS.CARD_LIST]: lodashGet(response, 'fundList', []),
+                [ONYXKEYS.BANK_ACCOUNT_LIST]: bankAccounts,
+                [ONYXKEYS.CARD_LIST]: debitCards,
```

Comment:
Do we need a migration for this?

Reply:
I'm not sure.. I'll test tomorrow. It is possible we would need a migration for current accounts with onyx caches...

Reply:
Great, thanks for double-checking on that.